README.md

```diff
@@ -1,13 +1,11 @@
 # Project React Native App 📱
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This is my first project using React Native, yay! 🥳
```

Comment:
Great quote app Camilla, also: love your design! It's always so inspiring I believe, the colors are so pretty and nice for the eye 🥇